cli/defenseclaw/commands/cmd_setup.py

```
@@ -11340,3 +11343,131 @@ def _show_splunk_credentials(data_dir: str) -> None:
+        app.cfg.training.enabled = True
+        if not app.cfg.training.backend:
+            app.cfg.training.backend = "mlx-lm-lora" if platform.system() == "Darwin" else "unsloth"
+
+        click.echo()
+        click.echo("  Setting up training pipeline...")
+        click.echo()
+
+        # 1. Install training backend
+        backend = app.cfg.training.backend
+        if backend == "mlx-lm-lora":
+            already = shutil.which("mlx-lm-lora") or _sp.run(
```

Ruff (0.16.1)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)